backend/Infrastructure/Repository.cs

```diff
@@ -17,7 +17,24 @@ public Repository(AppDbContext appDbContext)
         {
             _appDbContext.Add(entity);
         }
-
+        public void Delete<T>(T entity) where T : class
+        {
+            _appDbContext.Remove(entity);
+        }
+        public T GetById<T>(int id) where T : class
+        {
+            var entity = _appDbContext.Set<T>().Find(id);
+            if (entity == null)
```

Comment:
if entity was not found then it should return null

Reply:
what is this logic?
if entity is empty you can return null without checking ;p
simple
return _appDbContext.Set<T>().Find...

Reply:
in order to not show warning please change declaration from 
public T ...
to
public T? ...